src/DistortableCollection.js

```diff
@@ -243,9 +299,89 @@ L.DistortableCollection = L.FeatureGroup.extend({
     var layersToMove = this._calcCollectionFromPoints(cpd, overlay);
 
     layersToMove.forEach(function(layer) {
-      layer._updateCornersFromPoints(layer._cpd);
+      layer._setCornersFromPoints(layer._cpd);
       layer.fire("update");
     }, this);
+  },
+
+  _getAvgCmPerPixel: function(imgs) {
+    var reduce = imgs.reduce(function(sum, img) {
+      return sum + img.cm_per_pixel;
+    }, 0);
+    return reduce / imgs.length;
+  },
+
+  generateExportJson: function() {
+    var json = {};
+    json.images = [];
+
+    this.eachLayer(function(layer) {
+      if (this.isSelected(layer)) {
+        json.images.push({
+          id: this.getLayerId(layer),
+          src: layer._image.src,
+          nodes: layer.getCorners(),
+          cm_per_pixel: L.ImageUtil.getCmPerPixel(layer)
+        });
+      }
+    }, this);
+
+    json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
+
+    return json;
+  },
+
+  startExport: function(opts) {
+    opts = opts || {};
+    opts.collection = opts.collection || this.generateExportJson();
+    opts.frequency = opts.frequency || 3000;
+    opts.scale = opts.scale || 30;
+    var statusUrl, updateInterval;
+
+      // this may be overridden to update the UI to show export progress or completion
+    function _defaultUpdater(data) {
+      if (data.status === "complete") { clearInterval(updateInterval); }
+      // TODO: update to clearInterval when status == "failed" if we update that in this file:
+      // https://github.com/publiclab/mapknitter-exporter/blob/main/lib/mapknitterExporter.rb
+      console.log(data);
+    }
+
+
+    // receives the URL of status.json, and starts running the updater to repeatedly fetch from status.json; 
+    // this may be overridden to integrate with any UI
+    function _defaultHandleStatusUrl(data) {
+      console.log(data);
+      statusUrl = "//export.mapknitter.org" + data;
+      opts.updater = opts.updater || _defaultUpdater;
+
+      // repeatedly fetch the status.json
+      updateInterval = setInterval(function intervalUpdater() {
```

Comment:
Here I realized we made a mistake and were running the interval /only/ on the display of status, but not actually fetching the status repeatedly. We were just showing the same value over and over without fetching a new copy from the server!

We need to rebuild files over this change (i made the change directly to your PR, hope that's OK!)